qns/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 ENV RUSTUP_HOME=/usr/local/rustup \
     CARGO_HOME=/usr/local/cargo \
     PATH=/usr/local/cargo/bin:$PATH \
-    RUST_VERSION=1.45.2
+    RUST_VERSION=1.70.0
```

Comment:
Same as version in `check.yml`:

 https://github.com/mozilla/neqo/blob/6fb7a6cf1ffcc1af67f72313b59c05e1370b99d3/.github/workflows/check.yml#L20.